kitsune/customercare/cron.py

```diff
@@ -54,10 +54,14 @@ def collect_tweets():
 
     """Collect new tweets about Firefox."""
     with statsd.timer('customercare.tweets.time_elapsed'):
-        t = Twython(settings.TWITTER_CONSUMER_KEY,
-                    settings.TWITTER_CONSUMER_SECRET,
-                    settings.TWITTER_ACCESS_TOKEN,
-                    settings.TWITTER_ACCESS_TOKEN_SECRET)
+        t = Twython(
+            settings.TWITTER_CONSUMER_KEY,
+            settings.TWITTER_CONSUMER_SECRET,
+            settings.TWITTER_ACCESS_TOKEN,
+            settings.TWITTER_ACCESS_TOKEN_SECRET,
+            client_args={
+                'timeout': 10,
```

Comment:
I'd put a comment here if only to save someone else having to look it up. This means there's a 10 second connect timeout and a 10 second read timeout.

http://docs.python-requests.org/en/latest/user/advanced/?highlight=timeout#timeouts